features.json

```diff
@@ -208,6 +208,22 @@
 				"tailCall": false,
 				"threads": true
 			}
+		},
+		"wasm2c": {
+			"url": "https://github.com/WebAssembly/wabt",
+			"logo": "/images/wasm2c.svg",
+			"features": {
+				"bigInt": null,
+				"bulkMemory": true,
+				"exceptions": ["flag", "Requires flag `--enable-exceptions`"],
+			        "multiMemory": ["flag", "Requires flag `--enable-multi-memory`"],
+			        "multiValue": true,
```

Comment:
There's a mix of tabs and spaces here.

Reply:
Thanks, replaced with tabs to match rest of file.